src/app/shared/services/record.service.ts

```diff
@@ -31,29 +31,35 @@ import { AppConfig } from '../../app.config';
 export class RecordService {
   private pidType: string;
   private pidValue: string;
-  private config: AppConfig;
+  private objectId: string;
```

Comment:
An empty line above to separate from others. Also single comments to explain these properties would be nice.